### DISCUSSION TITLE
May i join the team?

### DISCUSSION THREAD

**Discussion by IdkAnythin07** [ROM-builders/temporary#19316] [General]
Github: https://github.com/IdkAnythin07
Telegram: https://t.me/IdkAnythin07

PR:
#18932 

Successful Builds:
https://cirrus-ci.com/github/ROM-builders/temporary/pull%2F18932

**Comment by IdkAnythin07** [comment]
@Apon77 sir?

**Reply by Apon77** [reply]
Please check #learn instruction in telegram group

**Reply by IdkAnythin07** [reply]
I already did learn git cli, i know how to commit, and somewhat know cherry pick
Also here's dt:
https://github.com/IdkAnythin07/android_device_realme_RMX1941
